hyperswitch/HyperViewModel.swift

```diff
@@ -61,6 +61,63 @@ class HyperViewModel: ObservableObject {
         })
         task.resume()
     }
+    
+    func preparePaymentManagementSheet() {
```

Comment:
why not create a generic network layer for both payment intent and ephemeral-key

Reply:
done. Created `NetworkUtils` for the same. 👍